lib/bulk/common.js

```diff
@@ -441,13 +444,519 @@ const BulkWriteError = function(error, result) {
 };
 util.inherits(BulkWriteError, MongoError);
 
+/**
+ * Add a single update document to the bulk operation
+ *
+ * @method
+ * @param {object} updateDocument update operations
+ * @param {class} bulk FindOperatorsOrdered or FindOperatorsUnordered
+ * @throws {MongoError}
+ * @return {OrderedBulkOperation or UnordedBulkOperation}
+ */
+const Update = function(updateDocument, bulk) {
```

Comment:
in general, we should do function declarations over function expressions. Here, it would be

```js
function Update(updateDocument, bulk) {
}
```